src/theory/strings/normal_form.cpp

```diff
@@ -135,6 +136,31 @@ void NormalForm::getExplanation(int index, std::vector<Node>& curr_exp)
   }
 }
 
+Node NormalForm::collectConstantStringAt(size_t& index)
+{
+  std::vector<Node> c;
+  while (d_nf.size() > index && d_nf[index].isConst())
+  {
+    c.push_back(d_nf[index]);
+    index++;
+    // break;
```

Comment:
What is this break comment?

Reply:
Ah, I've moved the code here from the rewriter and didn't change that. Fixed.